tests/check/test_stream.c

```diff
@@ -192,6 +200,10 @@ test_stream_item_create_from_type(char type, uint8_t id, SignedVideoCodec codec)
       bu_data = codec == SV_CODEC_H264 ? p_nalu_h264
                                        : (codec == SV_CODEC_H265 ? p_nalu_h265 : invalid_av1);
       break;
+    case 'O':
+      bu_data = codec == SV_CODEC_H264 ? oms_sei_nalu_h264
+                                       : (codec == SV_CODEC_H265 ? oms_sei_nalu_h265 : sei_av1);
```

Comment:
Right now you use a dummy SEI for AV1, which is the same as `Z`. Is that the best option? Or should it be NULL or something else?